README.md

```diff
@@ -1,2 +1,2 @@
 # test
```

Comment:
F401 'test' imported but unused

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/)** We accept bounties via [Bountysource](https://www.bountysource.com/).